cider/exec/template/StringOpsIR.cpp

```diff
@@ -326,10 +385,8 @@ std::unique_ptr<CodegenColValues> CodeGenerator::codegenStringOpExpr(
 
   CHECK_GE(expr->getArity(), 1UL);
   CHECK(expr->hasNoneEncodedTextArg());
-
   const auto& return_ti = expr->get_type_info();
-  // Should probably CHECK we have a UOper cast to dict encoded to be consistent
-  const auto primary_arg = remove_cast(expr->getArg(0));
+  const auto primary_arg = expr->getArg(0);
```

Comment:
I see in codegenPerRowStringOper, there is still a ```remove_cast(expr->getArg(0));```. Should ```remove_cast``` in there be removed too?

Reply:
`codegenPerRowStringOper` old codepath for string op, afraid it will affect old string_dict encoding, so keep it.